Fabric/src/main/java/me/ferdz/placeableitems/block/PlaceableItemsBlock.java

```diff
@@ -112,14 +113,6 @@ public PlaceableItemsBlock register(String name, Item item) {
         return this.register(name);
     }
 
```

Comment:
Okay, and it was too much maintenance for the point of limiting the user. Will always allow any placement now